app/services/hyrax/migrator/services/admin_set_membership_service.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rdf'
+
+module Hyrax::Migrator::Services
+  # A service to pull ids out of set-related fields for admin_set and collection memberships
+  class AdminSetMembershipService
+    def initialize(work, migrator_config)
+      @work = work
+      @config = migrator_config
+    end
+
+    def acquire_set_ids
+      result = {}
+      result['admin_set_id'] = admin_set(@work.env[:attributes])
```

Comment:
I think we might need a crosswalk sort of concept for the adminset and collections so that the original names from OD1 can be associated to a system ID in OD2 .. the only adminset that has a special ID is `admin/default`, while all of the others are going to be typical autogenerated IDs in Hyrax (ie. fe9f9e73).